Examples/Footprint_for_IR_list_L2.py

```diff
@@ -0,0 +1,246 @@
+
+IR_list=['1','5']
+#IR_list=['5']
+#IR_list=['1','5','2','8']
+
+OnlyHo=False
+OnlyLR=False
+############################
+
+import numpy as np
+import sys
+import xtrack as xt
+
+sys.path.append('../')
+
+import matplotlib.pyplot as plt
+
+import BBStudies.Physics.Detuning_L2 as dtuneL2
+from scipy import constants
+
+import BBStudies.Physics.Base as phys
+import BBStudies.Plotting.BBPlots as bbplt
+import BBStudies.Physics.Constants as cst
+import scipy.stats as sciStat
+from multiprocessing import Pool
+import time
+
+## %%
+my_json = ('../BBStudies/Run3_configuration/'
+           '2_configure_and_track/'
+           'final_collider.json')
+collider = xt.Multiline.from_json(my_json)
+collider.build_trackers()
+## %%
+
+
+## %%
+# Here a_min = 1 so that ERR=1.e-5 in Detuning_L2 is accurate enough
+# very small amplitudes min_a << 1 may require betterintegration accuracy
+a_min = .5
+coordinates = phys.polar_grid(r_sig=np.linspace(a_min, 6.5, 5),
+                              theta_sig=np.linspace(
+                                  0.05*np.pi/2, 0.95*np.pi/2, 6),
+                              emitt=[2.5e-6/7000, 2.5e-6/7000])
+ax = coordinates['x_sig']
+ay = coordinates['y_sig']
+
+energy = 7000
+# gamma relativistic of a proton at 7 TeV
+gamma_rel = energy / \
+    (constants.physical_constants['proton mass energy equivalent in MeV'][0]/1000)
+# beta relativistic of a proton at 7 TeV
+beta_rel = np.sqrt(1-1/gamma_rel**2)
+emittance_strong_nx = 2.5e-6
+emittance_strong_ny = 2.5e-6
+emittance_weak_nx = 2.5e-6
+emittance_weak_ny = 2.5e-6
+emittance_strong_x = emittance_strong_nx/gamma_rel/beta_rel
+emittance_strong_y = emittance_strong_ny/gamma_rel/beta_rel
+emittance_weak_x = emittance_weak_nx/gamma_rel/beta_rel
+emittance_weak_y = emittance_weak_ny/gamma_rel/beta_rel
+
+
+# Compute dummy xi
+Nb = 1e11
+round_emitt = 2.5e-6
+xi = Nb*cst.r_p/(4*np.pi*round_emitt)
+number_of_ho_slices = 11
+
+
+
+## %%
+#UseModel='IW'
+def DQx_DQy_L2(bb_name,  r,
+                dx_sig,
+                dy_sig,
+                A_w_s,
+                B_w_s,
+                xi):
+   return {bb_name:  (dtuneL2.DQx_DQy(coordinates['x_sig'],
+                                      coordinates['y_sig'],
+                                      dx_sig,
+                                      dy_sig,
+                                      A_w_s,
+                                      B_w_s,
+                                      r,
+                                      xi))}
+
+
+
+def Get_Footprint(IR):
```

Comment:
This function of string IR = '1' , '5',.. does the main calc.
Could not do it some other way eg a loo,  because string IR participates in survey  element0 = ...